student-store-express-api/app.js

```diff
@@ -1 +1,57 @@
-// YOUR CODE HERE
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const Store = require("./models/Store");
+const db = require("./data/db.json");
+const morgan = require("morgan");
+
+const errorHandler = (error, request, response, next) => {
+  // Error handling middleware functionality
+  console.error(error.message); // log the error
+  const status = error.statusCode || 500; // 500 because if no status code, the server is likely down
+  // send back an easily understandable error message to the caller
+  response.status(status).send(error.message);
+  next(error);
+};
+
+app.use(cors());
+app.use(express.json());
+app.use(errorHandler);
```

Comment:
This line isn't necessary as the error handler isn't a middleware and is more of a catchall route.